### ISSUE TITLE
Merge data mgmt and metadata mgmt?

### ISSUE THREAD

**Issue by alechenninger** [lightblue-platform/lightblue-applications#29]
Should the data management application and metadata management application be two separate applications? It seems like the activities of viewing and managing data and metadata and related enough that there could be benefits to this being the same application (AFAIK most RDBMS tools allow you to deal with both).

For instance, from the data mgmt application, it is likely you will want to view schema, or that the data mgmt application itself will have its own assistance with schema (autocomplete, etc). So either you have to hard code a link to a deployment to the metadata app, or you end up recreating some of the same functionality.

**Comment by paterczm** [comment]
I agree - a single, consistent looking tool would have been better. On the other hand I don't find it very important.
